devel/addon/config/ytt_values.yaml

```diff
@@ -0,0 +1,18 @@
+#! Copyright 2022 The cert-manager Authors.
+#!
+#! Licensed under the Apache License, Version 2.0 (the "License");
+#! you may not use this file except in compliance with the License.
+#! You may obtain a copy of the License at
+#!
+#!     http://www.apache.org/licenses/LICENSE-2.0
+#!
+#! Unless required by applicable law or agreed to in writing, software
+#! distributed under the License is distributed on an "AS IS" BASIS,
+#! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#! See the License for the specific language governing permissions and
+#! limitations under the License.
+
+#@data/values
+---
+gateway_ip:
```

Comment:
Eventually we could write the actual values in a values file like this once after the kind cluster has been created and then pass it to all targets that needs it.

Reply:
I actually removed these files now. I think I mixed up values files with schema file in ytt https://carvel.dev/ytt/docs/v0.40.0/how-to-write-schema/#starting-a-schema-document
We could have declared default values there, but weren't actually doing it, so it had no purpose.